src/host-config/types.ts

```diff
@@ -47,10 +57,12 @@ export type HostProxyConfig = t.TypeOf<typeof HostProxyConfig>;
  * with host-config PATCH and provided to the user with host-config GET.
  */
 export const HostConfiguration = t.type({
-	network: t.partial({
-		proxy: HostProxyConfig,
-		hostname: t.string,
-	}),
+	network: t.exact(
```

Comment:
Why change to `exact` here? This will fail validation if additional values are given no? I think the correct is to ignore additional properties.

Reply:
It actually won't fail validation, just strip additional properties from inputs. There's a test case to ensure this behavior [here](https://github.com/balena-os/balena-supervisor/pull/2284/files#diff-d3f5db585e205253dbabde1162d4c39c17be636d36c5a1354f7ae90583ff4cf3R993-R1026)

Reply:
Thanks for the clarification, I was confused because I've been using Zod on another project where `exact` is the standard and you need `strict` if you want validation to fail